### DISCUSSION TITLE
Managed server console cannot display Chinese

### DISCUSSION THREAD

**Discussion by tab960562** [Pryaxis/TShock#2924] [Q&A] [answered]
<img width="1022" alt="截圖 2023-03-07 下午5 31 55" src="https://user-images.githubusercontent.com/103242038/223382586-5e2ebbdc-3772-455e-94c5-7b464a51a729.png">

**Comment by sgkoishi** [comment, marked as answer]
> To change the locale in Linux, you can follow these steps:
> * Check the current locale settings by running the command:
> ```sh
> locale
> ```
> * To see a list of available locales, run:
> ```sh
> locale -a
> ```
> * Choose the desired locale from the list and set it by exporting the value of the LANG environment variable, for example, to set the locale to en_US.UTF-8, run:
> ```sh
> export LANG=en_US.UTF-8
> ```
> * You may also want to set other related environment variables, such as LC_CTYPE, LC_TIME, LC_NUMERIC, LC_COLLATE, LC_MONETARY, LC_MESSAGES, etc., depending on > your needs. For example, to set the time format to 24-hour clock, you can run:
> ```sh
> export LC_TIME=en_GB.UTF-8
> ```
> * If you want to make the changes permanent, you can edit the /etc/default/locale file and add the desired values there. For example:
> ```sh
> LANG=en_US.UTF-8
> LC_TIME=en_GB.UTF-8
> ```
> * After making the changes, you can either log out and log back in, or run the following command to apply the new settings:
> ```sh
> source /etc/default/locale
> ```
> Note: Depending on your Linux distribution and desktop environment, the steps above may vary slightly.

<sup>This answer is generated by [ChatGPT](https://openai.com/blog/chatgpt/) by OpenAI.</sup>
